src/instruments/src/SD/Pages/Hyd/Hyd.tsx

```diff
@@ -209,243 +123,329 @@ const RAT = ({ x, y }: RATProps) => {
-        switch (pump) {
-        case 'GREEN':
-            return greenPumpLowPressure || !greenPumpActive;
-        case 'BLUE':
-            return bluePumpLowPressure || !bluePumpActive;
-        case 'YELLOW':
-            return yellowPumpLowPressure || !yellowPumpActive;
-        default:
-            return 1;
-        }
-    }
+    const [pumpPressurisedSwitch] = useSimVar(`L:A32NX_HYD_${title}_PUMP_1_SECTION_PRESSURE_SWITCH`, 'boolean', 500);
+    const [systemPressurisedSwitch] = useSimVar(`L:A32NX_HYD_${title}_SYSTEM_1_SECTION_PRESSURE_SWITCH`, 'boolean', 500);
 
-    const pumpDetectLowPressure = checkPumpLowPressure(title);
+    const [reservoirLowQuantitySwitch] = useSimVar(`L:A32NX_HYD_${title}_RESERVOIR_LEVEL_IS_LOW`, 'boolean', 500);
 
-    const hydLevelBoolean = (value: boolean) => {
-        setHydLevelLow(value);
-    };
+    let hydTitleXPos: number;
+    if (title === 'GREEN') {
+        hydTitleXPos = -2;
+    } else if (title === 'BLUE') {
+        hydTitleXPos = 1;
+    } else {
+        hydTitleXPos = 3;
+    }
 
     return (
-        <>
-            <Triangle x={x} y={y} colour={pressureNearest50 <= lowPressure ? 'Amber' : 'Green'} fill={0} orientation={0} />
-            <text className={`Title ${pressureNearest50 <= lowPressure ? 'FillAmber' : 'FillWhite'}`} x={x} y={y + 43}>{title}</text>
-            <text className={`Pressure ${pressureNearest50 <= lowPressure ? 'FillAmber' : 'FillGreen'}`} x={x} y={y + 75}>{pressureNearest50}</text>
-
-            {/* The colour of these lines will be affected by the yellow electric pump */}
-            <line className={pressureNearest50 <= lowPressure ? 'AmberLine' : 'GreenLine'} x1={x} y1={y + 126} x2={x} y2={y + 83} />
-            <line
-                className={pressureNearest50 <= lowPressure
-                 || (pumpDetectLowPressure && title === 'GREEN')
-                 || (pumpDetectLowPressure && !yellowElectricPumpStatus && title === 'YELLOW') ? 'AmberLine' : 'GreenLine'}
-                x1={x}
-                y1={y + 181}
-                x2={x}
-                y2={y + 125}
-            />
-            <line className={pressureNearest50 <= lowPressure || (pumpDetectLowPressure && title !== 'BLUE') ? 'AmberLine' : 'GreenLine'} x1={x} y1={y + 221} x2={x} y2={y + 180} />
+        <SvgGroup x={x} y={y}>
+            <Triangle x={0} y={0} colour={!systemPressurisedSwitch ? 'Amber' : 'Green'} fill={0} orientation={0} />
+            <text className={`Huge Center${!systemPressurisedSwitch ? ' Amber' : ''}`} x={hydTitleXPos} y={50}>{title}</text>
+            <text className={`Huge Center ${pressureNearest50 <= lowPressure ? 'Amber' : 'Green'}`} x={1} y={92}>{pressureNearest50}</text>
+
+            <line className={pressureNearest50 <= lowPressure ? 'AmberLine' : 'GreenLine'} x1={0} y1={title === 'BLUE' ? 217 : 151} x2={0} y2={103} />
 
             <HydEngPump
                 system={title}
                 pumpOn={pumpPBStatus}
```

Comment:
Do we actually want to pass along the PB state or the pump state? The variable names are somewhat ambiguous. It looks like we are using `bluePumpPBStatus && bluePumpActive`.

Reply:
Yes this is somewhat unclear. The blue pump is a special case though, since the LO indication for it also depends on if the pump is actually running. I'll rename the parameters to make it clearer